src/Argu/Utils.fs

```diff
@@ -152,7 +172,7 @@ type UnionCaseInfo with
 
         match uci.GetCustomAttributes typeof<'T> with
         | [||] when includeDeclaringTypeAttrs ->
-            match uci.DeclaringType.GetCustomAttributes(typeof<'T>, false) with
+            match uci.DeclaringType.GetCustomAttributes(typeof<'T>, false) |> Seq.toArray with
```

Comment:
@enricosada I see your point, my feeling though is that such superficial changes cause more problems than what they allegedly solve.